.nvmrc

```diff
@@ -0,0 +1 @@
+v13.10.1
```

Comment:
In other apps we use `lts/*` instead of specifying an exact version here - is that a good idea here?

Reply:
TIL, will do!